frontend/src/features/map/layers/BaseLayer.tsx

```diff
@@ -1,75 +1,103 @@
-import { MapboxVector } from 'ol/layer'
 import TileLayer from 'ol/layer/Tile'
```

Comment:
IMO this file should be a (side effect) hook.

Reply:
Yep, all map-related components (map constructor, map layers, map event listeners) should be moved to singleton hooks.
I propose to do that after the current EPIC ?